src/test/http-mocks/claim-store.ts

```diff
@@ -236,11 +241,25 @@ export function rejectSaveOfferForDefendant (reason: string = 'HTTP error') {
     .post(new RegExp('/[0-9]+/offers/defendant'))
     .reply(HttpStatus.INTERNAL_SERVER_ERROR, reason)
 }
+
 export function resolveSaveOffer () {
   mock(`${serviceBaseURL}/claims`)
     .post(new RegExp('/[0-9]+/offers/defendant'))
-    .reply(HttpStatus.OK)
+    .reply(HttpStatus.CREATED)
 }
+
+export function resolveAcceptOffer (by: string = 'claimant') {
```

Comment:
Should't that be in capitals? Would be possible to reuse `MadeBy` buy repurposing the class so we have `OfferParticipantType.CLAIMANT.value`. I am pretty sure we could find better name as well :)

Reply:
It doesn't matter if it's upper or lower case. Both works for the same endpoint.

This is actually weird we don't have consts like this so far on frontend. If we want to introduce it what do you think about MadeBy enum (as we have one in API)? The name is more generic and we can use it in more contexts.